Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -14,8 +14,24 @@ public enum PlayerSpawnMethod
     }
 
     [AddComponentMenu("Network/NetworkManager")]
+    [RequireComponent(typeof(Transport))]
     public class NetworkManager : MonoBehaviour
     {
+        // transport layer
+        // -> automatically uses the first transport component. there might be
+        //    multiple in case of multiplexing, so the order matters.
+        Transport _transport;
+        public Transport transport
```

Comment:
Make this depend on an ITransport interface instead

Reply:
Make this virtual,   people might want to override this and customize transport initialization